script/test

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
+# vim:ts=2:sw=2:et
```

Comment:
Could you 🔥  this vim comment? I just prefer not to have editor-specific stuff in the code.

Reply:
Sure.